UpdateProfile/__tests__/handler.test.ts

```diff
@@ -847,4 +845,76 @@ describe("UpdateProfileHandler", () => {
 
     expect(mockSendMessage).toBeCalledTimes(0);
   });
+
+  it("GIVEN a profile with a valid last_app_version, the handler should write the field and return successfully", async () => {
+    const profileModelMock = {
+      findLastVersionByModelId: jest.fn(() => TE.of(some({ ...aRetrievedProfile, lastAppVersion: undefined }))),
```

Comment:
This should never happen. The decode of the `findLastVersionByModelId` model method adds the default value.